src/inc/aduc/result.h

```diff
@@ -2119,6 +2124,21 @@ MAKE_ADUC_EXTENDEDRESULTCODE_FOR_FACILITY_ADUC_FACILITY_UNUSED_F(const int32_t c
 #define ADUC_ERC_DOWNLOAD_HANDLER_EXTENSIBILITY_GET_CONTRACT \
     MAKE_ADUC_EXTENDEDRESULTCODE_FOR_COMPONENT_ADUC_COMPONENT_DOWNLOAD_HANDLER_EXTENSION_MANAGER(5)
 
+/**
+ * @brief ADUC_ERC_DOWNLOAD_HANDLER_CREATE_FACTORY_INSTANCE, ERC Value: 2415919107 (0x90000003)
+ */
+ #define ADUC_ERC_DOWNLOAD_HANDLER_CREATE_FACTORY_INSTANCE MAKE_ADUC_EXTENDEDRESULTCODE_FOR_COMPONENT_ADUC_COMPONENT_DOWNLOAD_HANDLER_EXTENSION_MANAGER(3)
```

Comment:
Remove? DUP of line 2112-2125.
Probably better off building clean and doing git add on generated result.h

Reply:
How is this getting duped? It should be generated by the code never touched by us. There should be no way for there to be a duplicate

Reply:
> How is this getting duped? It should be generated by the code never touched by us. There should be no way for there to be a duplicate
@nihemstr maybe it was somehow a bad merge of checked-in result.h, which is why I mentioned to regenerate it.